### ISSUE TITLE
Pitcher Names

### ISSUE THREAD

**Issue by kickinbahk** [jaw187/mlbprobablepitchers#1]
Thank you for this module. 

I was wondering the best way to get the names of the pitchers rather than the player id. Is there a specific reason you are getting the id instead? 

This is a sample of the json I am currently getting for a single game: 

```
 {
    id: '2016/04/06/chamlb-oakmlb-1',
    startTime: '2016-04-06T19:05:00',
    easternTime: '2016-04-06T22:05:00',
    timezone: 'PDT',
    teams: { away: '133', home: '145' },
    pitchers: { away: '607074', home: '605135' } 
}
```

**Comment by jaw187** [comment]
@kickinbahk I might be able to include the name.  At the moment, I do all of my work with IDs that I use to compare against a database of player details which I seed from http://crunchtimebaseball.com/baseball_map.html

But I'm sure most folks won't want to have to do that.  And since the MLB.com page already displays the pitchers name, it shouldn't be much work for me to include, so I'm going to consider this a feature request.

**Comment by jaw187** [comment]
https://github.com/jaw187/mlbprobablepitchers/commit/6930f06309818b644466161b2c9e4903612d1a4d contains the update to include new details, but something is wrong with lab and my test isn't running correctly.  I'm not going to publish a new version until I straighten that out.

**Comment by jaw187** [comment]
Test is not a problem now, not sure what happened there....

@kickinbahk just published version 1.0.0, go ahead and check that out, you can expect a payload which looks similar to 

``` javascript
{
    "id": "2011/07/23/houmlb-chnmlb-1",
    "startTime": "2011-07-23T12:05:00",
    "easternTime": "2011-07-23T13:05:00",
    "timezone": "CDT",
    "teams": {
        "away": "112",
        "home": "117"
    },
    "pitchers": {
        "away": {
            "id": "434643",
            "name": "Wandy Rodriguez",
            "throws": "LHP"
        },
        "home": {
            "id": "448694",
            "name": "Randy Wells",
            "throws": "RHP"
        }
    }
}
```
